core/gui/src/app/dashboard/component/user/user-workflow/user-workflow-list-item/user-workflow-list-item.component.ts

```diff
@@ -22,8 +23,8 @@ import { DownloadService } from "src/app/dashboard/service/user/download/downloa
   styleUrls: ["./user-workflow-list-item.component.scss"],
 })
 export class UserWorkflowListItemComponent {
-  ROUTER_WORKFLOW_BASE_URL = "/dashboard/workspace";
-  ROUTER_USER_PROJECT_BASE_URL = "/dashboard/user-project";
+  ROUTER_WORKSPACE_BASE_URL = DASHBOARD_USER_WORKSPACE;
+  ROUTER_USER_PROJECT_BASE_URL = DASHBOARD_USER_PROJECT;
```

Comment:
do we have to rename them? or we can just use the same variable imported?

Reply:
also, if declare such constant variable, maybe better to put it outside the class and make it export.

Reply:
I found that in the Angular framework, if we want to use constants in the HTML template, we must declare them inside the class, even if they are already defined outside the class. However, it is possible to avoid redeclaring constants in the class by calling a TypeScript method from the HTML template and referencing the constants from app-routing.constant within the method. What do you think of this method?

Reply:
not a good idea to call a function in html as the function will be evaluated everytime when it needs to check for rendering. using a variable is better. in this case, yes we can keep the re-declaration to make the imported varible as a class variable for html to use. But please keep the names the same, don't change to a different name.